### PR TITLE
:sparkles: Accessibility > colorblindness shaders

### PR DESCRIPTION
**Pull request by Tugduoff**
Added shaders for : 
- Deuteranopia
- Protanopia
- Tritanopia

### CODE REVIEW

**Review by C0Florent — DISMISSED** [review]
Seems all good to me, apart from the shared pointer but honestly we have other priorities, this is all fine

**Review by Tholarr — DISMISSED** [review]
Noice :+1:

**Review by ClementBaziret — DISMISSED** [review]
Seems all good to me :saluting_face:

File: src/client/configClient.cfg [hunk 1]
```diff
@@ -29,6 +29,7 @@ systems = (
             title = "RType",
             texturesPath = "./config/textures.cfg",
             shadersPath = "./config/shaders.cfg",
+            defaultShader = "protanopia",
```
Comment (ClementBaziret) [inline src/client/configClient.cfg:32]: default value should not be a filter I think

**Review by ClementBaziret — COMMENTED** [review]
Just the default value to change

**Review by ClementBaziret — APPROVED** [review]
Seems all good to me :saluting_face:

File: src/client/configClient.cfg (hunk 1 above)
Comment (Tugduoff) [reply src/client/configClient.cfg]: Indeed, it has been set back to default

**Review by Tholarr — APPROVED** [review]
:+1:

**Review by C0Florent — APPROVED** [review]

**Pull request merged**